docs/migration-guide-from-v4-to-v5.md

```diff
@@ -0,0 +1,37 @@
+Migration guide from v4 to v5
+=============================
+
+We're happy to announce that Sharpbrake version 5 brings support for error messages along with exceptions. This functionality comes with changes to our API and here are steps to migrate to the new version:
+
+1. Use the `BuildNotice` method instead of `NotifyAsync` to pass `exception` object. The `SetHttpContext` method of `notifier` can be used to set `HttpContext`. Call `NotifyAsync` with `notice` returned from `BuildNotice`.
+
+   Before:
+
+   ```csharp
+   notifier.NotifyAsync(exception, httpContext, severity);
+   ```
+
+   After:
+
+   ```csharp
+   var notice = notifier.BuildNotice(severity, exception);
+   notifier.SetHttpContext(notice, httpContext);
+   notifier.NotifyAsync(notice);
+   ```
+
+2. The `Notify` method was removed. It was a convenience method around `NotifyAsync` that doesn't bring any real value. The same functionality can be achieved using `NotifyAsync` + continuation:
+
+   ```csharp
+   notifier.NotifyAsync(notice).ContinueWith(responseTask =>
+   {
+       if (responseTask.IsFaulted)
+           Console.WriteLine("Response task is faulted");
+       else
+       {
```

Comment:
Style question: why do we not omit brackets here, but omit them for the line above?

Reply:
Missing brackets in the first branch implies that this is completed logic (`responseTask` is faulted - nothing can be done with it so just report on the issue and stop) while the second branch (responseTask is Okay) encourage for some additional processing that will span across multiple lines of code ;)